chapters/hpmor-chapter-002.tex

```diff
@@ -1,114 +1,291 @@
+}{%
+}
 
+\iftoggle{isEnglish}{%
 Harry scrambled back unthinkingly, back-pedalling so fast that he tripped over a stray stack of books and landed hard on his bottom with a \emph{thwack}. His hands came down to catch himself without quite reaching properly, and there was a warning twinge in his shoulder as the weight came down unbraced.
+}{%
+}
 
+\iftoggle{isEnglish}{%
 At once the small tabby cat morphed back up into a robed woman. “I’m sorry, Mr~Potter,” said the witch, sounding sincere, though the corners of her lips were twitching upwards. “I should have warned you.”
+}{%
+}
 
+\iftoggle{isEnglish}{%
 Harry was breathing in short gasps. His voice came out choked. “\emph{You can’t \emph{do} that!}”
+}{%
+}
 
+\iftoggle{isEnglish}{%
 “It’s only a Transfiguration,” said Professor McGonagall. “An Animagus transformation, to be exact.”
+}{%
+}
 
+\iftoggle{isEnglish}{%
 “You turned into a cat! A \emph{small} cat! You violated Conservation of Energy! That’s not just an arbitrary rule, it’s implied by the form of the quantum Hamiltonian! Rejecting it destroys unitarity and then you get FTL signalling! And cats are \emph{complicated}! A human mind can’t just visualise a whole cat’s anatomy and, and all the cat biochemistry, and what about the \emph{neurology}? How can you go on \emph{thinking} using a cat-sized brain?”
+}{%
+}
 
+\iftoggle{isEnglish}{%
 Professor McGonagall’s lips were twitching harder now. “Magic.”
+}{%
+}
 
+\iftoggle{isEnglish}{%
 “Magic \emph{isn’t enough} to do that! You’d have to be a god!”
+}{%
+}
 
+\iftoggle{isEnglish}{%
 Professor McGonagall blinked. “That’s the first time I’ve ever been called \emph{that}.”
+}{%
+}
 
+\iftoggle{isEnglish}{%
 A blur was coming over Harry’s vision, as his brain started to comprehend what had just broken. The whole idea of a unified universe with mathematically regular laws, that was what had been flushed down the toilet; the whole notion of \emph{physics}. Three thousand years of resolving big complicated things into smaller pieces, discovering that the music of the planets was the same tune as a falling apple, finding that the true laws were perfectly universal and had no exceptions anywhere and took the form of simple maths governing the smallest parts, \emph{not to mention} that the mind was the brain and the brain was made of neurons, a brain was what a person \emph{was}—
+}{%
+}
 
+\iftoggle{isEnglish}{%
 And then a woman turned into a cat, so much for all that.
+}{%
+}
 
+\iftoggle{isEnglish}{%
 A hundred questions fought for priority over Harry’s lips and the winner poured out: “And, and what kind of incantation is \emph{Wingardium Leviosa?} Who invents the words to these spells, nursery schoolers?”
+}{%
+}
 
+\iftoggle{isEnglish}{%
 “That will do, Mr~Potter,” Professor McGonagall said crisply, though her eyes shone with suppressed amusement. “If you wish to learn about magic, I suggest that we finalise the paperwork so that you can go to Hogwarts.”
+}{%
+}
```

Comment:
```suggestion
}{%
“That will do, Mr~Potter,” Professor McGonagall said crisply, though her eyes shone with suppressed amusement.
“If you wish to learn about magic, I suggest that we finalise the paperwork so that you can go to Hogwarts.”
}
```